src/noshow/database/models.py

```diff
@@ -37,6 +37,7 @@ class ApiPatient(Base):
         index=True,
     )
     call_number: Mapped[int] = mapped_column(Integer, init=False, nullable=True)
+    last_call_date: Mapped[datetime] = mapped_column(DateTime, nullable=True)
```

Comment:
It's a date right? Not datetime?

Reply:
Also this should contain `init=False` otherwise you have to include it when creating a Patient (also see failing unit test)